systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -462,6 +465,63 @@ public void prepareEnvForOperator(ExtensionContext extensionContext, String clie
         prepareEnvForOperator(extensionContext, clientNamespace, Collections.singletonList(clientNamespace));
     }
 
+    private String changeLeaseNameInResourceIfNeeded(String yamlPath) {
+        final EnvVar leaseEnvVar = extraEnvVars.stream().filter(envVar -> envVar.getName().equals("STRIMZI_LEADER_ELECTION_LEASE_NAME")).findFirst().orElse(null);
+        if (leaseEnvVar != null && yamlPath.contains("022-")) {
```

Comment:
Wouldn't be better to create a list of resource names and test path against it instead of this magic constant?